backend/tests/v5/unit/models/users.test.js

```diff
@@ -241,6 +241,13 @@ const testDeleteFromFavourites = () => {
 			expect(fn.mock.calls.length).toBe(0);
 		});
 
+		test('Should not return error when trying to remove all favourites from a specific teamspace but the user doesn\'t have any', async () => {
```

Comment:
It will still throw an error if the user has no favourites right ? It will not throw error if favouritesToRemove param is empty. Maybe the description needs an adjustment ?

Reply:
it should throw an error if i'm trying to remove a specific favourite (i.e. provided a fed/model id) - this is a separate existing test.

it shouldn't if i am trying to remove any favourites within a certain teamspace (that's the change i made within the codebase)

I think this describe accurately described the situation?

Reply:
Right I see.. However the findOne mock is still returning some favourites. Shouldnt it be mocked to return null ?

Reply:
doesn't really matter , the teamspace in question (`teamspace3`) has no entry on the data we mocked (teamspace1 and teamspace2).

updated as you wish